profile/latest/advisor/management/advisor/models.go

```diff
@@ -0,0 +1,40 @@
+package advisor
+
+import (
+	 original "github.com/Azure/azure-sdk-for-go/service/advisor/management/2017-04-19/advisor"
+)
+
+type (
+	 Category = original.Category
```

Comment:
So, it's hard to say because golint hasn't been updated to handle type aliases yet. At the moment it fails. However, I'm pretty certain that the stuttering shouldn't be triggered here. Even if it was, the value of having the type names be the same is high enough we would ignore those violations.

In terms of comments, it would be pretty easy to modify my tool to pull in the original comments. I'll make sure that happens for before this hits master.